listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -14,6 +14,28 @@
 from listenbrainz.db.exceptions import DatabaseException
 from spotipy import SpotifyException
 from werkzeug.exceptions import BadRequest, InternalServerError, ServiceUnavailable
+from brainzutils.mail import send_mail
+from brainzutils import musicbrainz_db
+from brainzutils.musicbrainz_db import editor as mb_editor
+
+
+def notify_error(musicbrainz_row_id, error):
+    """ Notifies specified user via email about error during Spotify import.
+
+    Args:
+        musicbrainz_row_id (int): the MusicBrainz row ID of the user
+        error (str): a description of the error encountered.
+    """
+    user_email = mb_editor.get_editor_by_id(musicbrainz_row_id)['email']
+    text = ('Hi, we encountered an error while importing your listens from Spotify.'
```

Comment:
I typically don't use multiline strings like this -- instead I'd use `"""`. However, as an alternative we could use jinja here and have a separate template file for the email. It might be nice to make it more consistent like other templates

Reply:
Changed to use jinja